### PR TITLE
feature/SOF-7166 Add tutorial for Python Transformation

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Create an Interface with Python Transformation
+
+In this tutorial, you will learn how to create an interface between two materials using the Python Transformation feature. Specifically, we will explore the creation of an interface between Graphene and Ni(111).
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:3]: In this tutorial, we create an interface

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Open Materials Designer
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:5]: Step 0: Open Materials Designer

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+We start with [opening](../../entities-general/actions/create.md) an instance of the [Materials Designer Interface](../../materials-designer/overview.md) for creating and designing new [Materials structures](../../materials/overview.md) on our platform.
+
+## Import Materials from Standata
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:9]: Step 1: Import Materials

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+<img src="/images/tutorials/interface_with_python/open_standata.png" alt="Open Import from Standata Dialog"/>
+- Select 'Graphene' and 'Ni' from the materials list.
+<img src="/images/tutorials/interface_with_python/import_from_standata.png" alt="Import Gr and Ni from Standata"/>
+- Click 'Submit' to import these materials into the Materials Designer session.
+- Default Material Silicon should be removed from the session.
+<img src="/images/tutorials/interface_with_python/remove_silicon.png" alt="Remove Silicon"/>
+- Graphene and Ni should now be available in the materials list.
+<img src="/images/tutorials/interface_with_python/graphene_and_ni_imported.png" alt="Gr and Ni available in materials list"/>
+
+## Use Python Transformation Dialog
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:25]: Step 2: Use Python Transformation Dialog

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        # fmt: on
+    },
+    "interface": {
+        "distance": 3.0,
+    },
+    # If True the layer cell and basis vectors will be scaled to fit the substrate cell.
+    # Mind the strain that is introduced by this operation.
+    "scale_layer_to_fit": False,
+}
+```
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:88]: Let's remove the settings - it's already in the code. Maybe add a screenshot instead, if not there

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- Click `Run All` to process the transformation.
+- The strain matrix will appear in the output, providing insight into the lattice deformation.
+- `Output Materials` will update with the newly created structure.
+<img src="/images/tutorials/interface_with_python/after_run.png" alt="Results of code execution: strain matrix and output materials"/>
+
+- Review the resulting strain matrix, if satisfied, submit the form to add the interface to your materials collection.
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:94]: This needs a picture with resulting material

File: lang/en/docs/tutorials/materials/interface-with-python.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## **Key Considerations**
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/interface-with-python.md:96]: Move to the top and use !note (see examples in other pages)

File: lang/en/docs/tutorials/materials/interface-with-python.md (hunk 5 above)
Comment (VsevolodX) [reply lang/en/docs/tutorials/materials/interface-with-python.md:88]: Screenshot won't fit all settings for that screen size.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**